src/components/eth-wallet/eth-wallet.tsx

```diff
@@ -85,67 +88,65 @@ const ConnectedKey = () => {
 
   return (
     <>
-      <WalletCardRow
-        label={t("vegaInWallet", { symbol: "$VEGA" })}
-        value={totalInWallet}
-        dark={true}
-        valueSuffix={t("VEGA")}
+      <WalletCardAsset
+        image={vegaVesting}
+        decimals={appState.decimals}
+        name="VEGA"
+        symbol="In vesting contract"
+        balance={totalInVestingContract}
       />
-      {Flags.STAKING_DISABLED ? null : (
-        <WalletCardRow
-          label={t("Not Associated")}
-          value={walletBalance}
-          valueSuffix={t("VEGA")}
-        />
-      )}
-      {Flags.STAKING_DISABLED ? null : (
-        <WalletCardRow
-          label={t("Associated")}
-          value={lien}
-          valueSuffix={t("VEGA")}
-        />
-      )}
-      <hr style={{ borderColor: Colors.BLACK, borderTop: 1 }} />
       {Flags.REDEEM_DISABLED ? null : (
         <>
-          <WalletCardRow
-            label={t("VESTING VEGA TOKENS")}
-            valueSuffix={t("VEGA")}
-            dark={true}
-            value={totalInVestingContract}
-          />
-          <WalletCardRow
-            label={t("Locked")}
-            value={totalLockedBalance}
-            valueSuffix={t("VEGA")}
-          />
-          <WalletCardRow
-            label={t("Unlocked")}
-            value={totalVestedBalance}
-            valueSuffix={t("VEGA")}
-          />
+          <WalletCardRow label={t("Locked")} value={totalLockedBalance} />
+          <WalletCardRow label={t("Unlocked")} value={totalVestedBalance} />
         </>
       )}
+      <WalletCardAsset
+        image={vegaWhite}
+        decimals={appState.decimals}
+        name="VEGA"
+        symbol="In Wallet"
+        balance={totalInWallet}
+      />
+      {Flags.STAKING_DISABLED ? null : (
+        <WalletCardRow label={t("Not Associated")} value={walletBalance} />
+      )}
+      {Flags.STAKING_DISABLED ? null : (
+        <WalletCardRow label={t("Associated")} value={lien} />
+      )}
       {Flags.STAKING_DISABLED || Flags.REDEEM_DISABLED ? null : (
         <>
           {associationsByVegaKey.length ? (
             <>
               <hr style={{ borderStyle: "dashed", color: Colors.TEXT }} />
```

Comment:
can we put the inline styles in a class?

Reply:
This is removed in a follow up PR to this one, so I will leave it for now to avoid conflicts, have removed all other instances of style in this file :D